cmd/collector/main.go

```diff
@@ -97,10 +101,19 @@ func run(log logrus.FieldLogger) error {
 	} else {
 		conntracker, err = conntrack.NewNetfilterClient(log)
 	}
-
 	if err != nil {
 		return err
 	}
+
+	var ip2dns dns.DNSLookup = &dns.Noop{}
+	if ebpf.IsKernelBTFAvailable() {
+		tracer := ebpf.NewTracer(log, ebpf.Config{
+			// TODO: Configurable
```

Comment:
does this comment mean that it should be extended to be support more config options? :)